src/cloudai/cli/handlers.py

```
@@ -303,6 +304,12 @@ def handle_dry_run_and_run(args: argparse.Namespace) -> int:
         return 1
     system, test_scenario, tests = setup_result
 
+    try:
+        validate_dse_env_params(test_scenario)
+    except TestScenarioParsingError as e:
+        logging.error(str(e))
```

Ruff (0.15.18)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)